### PR TITLE
[MARVP-5] Integrate Linkedin Login

### PR DESCRIPTION
**Pull request by vijaysamanthapudi**
Related: https://konysolutions.atlassian.net/browse/MARVP-5

### CODE REVIEW

File: KonyAccelerate/controllers/mobile/frmLoginController/frmLoginControllerActions.js [hunk 1]
```diff
@@ -3,13 +3,13 @@ define({
...
+    AS_UWI_f5b772abfeca4d809b68f66f91b61a75: function AS_UWI_f5b772abfeca4d809b68f66f91b61a75(response) {
         var self = this;
         var ntf = new kony.mvc.Navigation("frmEventList");
         ntf.navigate();
     },
     /** onLoginFailure defined for linkedinlogin **/
-    AS_UWI_j7f038a6352244d39e78302c0ecffcd0: function AS_UWI_j7f038a6352244d39e78302c0ecffcd0(response) {
+    AS_UWI_ge777c6098f545e693c0537e7f6d99e6: function AS_UWI_ge777c6098f545e693c0537e7f6d99e6(response) {
         var self = this;
         alert("login failed");
```
Comment (krishk95) [inline KonyAccelerate/controllers/mobile/frmLoginController/frmLoginControllerActions.js]: Please remove the alerts

**Review by krishk95 — APPROVED** [review]

**Review by mishradileep — CHANGES_REQUESTED** [review]
Do we need to push all the variants of Karbon fonts ? Please remove which are not in use

File: KonyAccelerate/userwidgets/com.konymp.linkedinlogin/userwidgetmodel.sm/flxMain.json [hunk 2]
```diff
@@ -23,20 +23,21 @@
         0
     ],
     "_percent_": true,
-    "_skin_": "s98231a0fe6f48d3b1fbe820c32218eb",
-    "_top_": {
-        "type": "string",
-        "value": "0%"
-    },
+    "_skin_": "s90dae5418f546c9bbfde0ff0a290743",
```
Comment (vigneshwarankarunanithi) [inline KonyAccelerate/userwidgets/com.konymp.linkedinlogin/userwidgetmodel.sm/flxMain.json:26]: Remove the blue border.

**Review by vigneshwarankarunanithi — COMMENTED** [review]
1. Fix Browser widget height issue (run time)

File: KonyAccelerate/controllers/mobile/frmLoginController/frmLoginControllerActions.js (hunk 1 above)
Comment (vijaysamanthapudi) [reply KonyAccelerate/controllers/mobile/frmLoginController/frmLoginControllerActions.js]: Alert Removed. UI will be replacing the alert.

File: KonyAccelerate/userwidgets/com.konymp.linkedinlogin/userwidgetmodel.sm/flxMain.json (hunk 2 above)
Comment (vijaysamanthapudi) [reply KonyAccelerate/userwidgets/com.konymp.linkedinlogin/userwidgetmodel.sm/flxMain.json:26]: Border Removed

**Pull request merged**